src/Task.ts

```diff
@@ -363,6 +384,13 @@ export class Task {
     public toString(layoutOptions?: LayoutOptions): string {
         layoutOptions = layoutOptions ?? new LayoutOptions();
         let taskString = this.description;
+        //Reminder can't be inshort mode, otherwise obsidian-reminder won't work!!!!
```

Comment:
The short mode is only ever used for displaying the text in Tasks results blocks.

When Tasks writes tasks out as Markdown, it always writes all fields.

Therefore, I think a new option to hide reminders should be added, for display purposes. I can do this as a separate pull request though.